node/src/Commands.ts

```diff
@@ -433,6 +433,29 @@ export function createHGet(
     return createCommand(RequestType.HGet, [key, field]);
 }
 
+/**
+ * @internal
+ */
+export function createHScan(
+    key: string,
+    cursor: string,
+    options?: BaseScanOptions,
+): command_request.Command {
+    const args: string[] = [key, cursor];
+
+    if (options) {
```

Comment:
I think we need a function, which converts `BaseScanOptions` to args array

Reply:
yup i've added it 👍